pkg/systemd/index.html

```diff
@@ -99,7 +99,7 @@
                 <span id="system_information_os_text"></span>
 
                 <div role="group" class="system-information-updates">
-                    <span id="system_information_updates_icon" hidden></span>
+                    <span id="system_information_updates_icon"></span>
```

Comment:
Was that deliberate? To avoid the text jumping around? This should be split out into a separate commit with justification (FTR, I agree, but it should be explicit).

Reply:
It's not necessary, the visibility is immediately determined by the code upon loading.